tests/meteor.yml

```diff
@@ -0,0 +1,31 @@
+---
+runner: command
+dir: meteor
+args:
+  - ../meteor-doc-fetcher.sh
+  - ../meteor-oplog-cursor-supported.sh
+  - ../meteor-oplog-entry-skipping.sh
+  - ../meteor-oplog-x-implicit-collection-creation.sh
+
+results:
+  postgresql:
+    stats:
+      fail: 4
+    fail:
+      - meteor-doc-fetcher.sh
+      - meteor-oplog-cursor-supported.sh
+      - meteor-oplog-entry-skipping.sh
+      - meteor-oplog-x-implicit-collection-creation.sh
+
+  sqlite:
+    stats:
+      fail: 4
+    fail:
+      - meteor-doc-fetcher.sh
+      - meteor-oplog-cursor-supported.sh
+      - meteor-oplog-entry-skipping.sh
+      - meteor-oplog-x-implicit-collection-creation.sh
+
+  mongodb:
+    stats:
+      pass: 4
```

Comment:
These are all failures at the moment because I need to configure the MongoDB service a single node replica set.

Reply:
Not sure how we can spin up a different environment per DB.

```
      - name: Start environment
        run: bin/task env-up-detach DB=${{ matrix.db }}
        env:
          FERRETDB_IMAGE: ghcr.io/ferretdb/ferretdb-dev:main
      
      - name: Start MongoDB environment
        if: ${{ matrix.db == 'mongodb' }}
        run: bin/task env-up-repl
```